css/styles.css

```diff
@@ -0,0 +1,329 @@
+  background-color: #f7dad9;
+  border-bottom: solid black 3px;
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  margin-right: 1.5rem;
+}
+
+.logo {
+  width: 65%;
+  margin: 0.8rem 0;
+  height: 20%;
+}
+
+.mobile-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  transition: 1s;
+}
+
+img {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.carousel-wrapper {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 220px;
+}
+
+.carousel {
+  margin-top: 1rem;
+  height: 200px;
+}
+
+.hidden {
+  display: none;
+}
+
+i {
+  color: #c15b75;
+  width: 100%;
+  display: flex;
+  line-height: normal;
+}
+
+.fa-magnifying-glass {
+  align-items: center;
+  justify-content: center;
+}
+
+.search {
+  width: 55vw;
+  border-radius: 0.5rem;
+  border-style: hidden;
+  border: none;
+  padding: 0.3rem 0;
+  font-size: 25px;
+  text-decoration: none;
+  height: 2rem;
+  margin-left: 1rem;
+}
+
+.search:hover {
+  cursor: pointer;
+}
+
+.search:focus {
+  outline: none;
+}
+
+.search-wrapper {
+  margin-top: 1.5rem;
+  margin-bottom: 2rem;
+  display: flex;
+  justify-content: center;
+  padding-left: 1rem;
+  background-color: white;
+  border: solid 2px black;
+  border-radius: 0.5rem;
+  align-items: center;
+}
+
+.spray-icon {
+  background-color: #f7dad9;
+  border: 3px solid black;
+  position: relative;
+  padding: 0.5rem 1.5rem;
+  border-radius: 1rem;
+}
+
+.card-container {
+  display: flex;
+  justify-content: center;
+  border-radius: 1rem;
+  margin: 0 4rem;
+  flex-wrap: wrap;
+  gap: 30px;
+  margin-bottom: 7rem;
+}
+
+.card-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: #87bbe1;
+  border: 3px solid black;
+  border-radius: 1rem;
+  margin-top: 1rem;
+  padding: 0.4rem;
+}
+
+.spray-title {
+  border-radius: 1rem;
+  border: 3px solid black;
+  background-color: #fdeaac;
+  text-align: center;
+  width: 100%;
+  align-items: center;
+  font-size: 23px;
+  margin-top: 1rem;
+}
+
+.text-wrapper {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+body {
+  background-image: linear-gradient(white, white, white, #88dbff);
+  height: 100%;
+  background-size: cover;
+  margin: 0;
+  background-attachment: fixed;
+}
+
+.sidenav {
+  height: 100%;
+  width: 50%;
+  position: fixed;
+  z-index: 1;
+  background-color: rgb(135, 187, 225, 0.8);
+  padding-top: 60px;
+  border: solid black 2px;
+  top: 0;
+  right: 0;
+}
+
+.sidenav .nav-button {
+  text-align: center;
+  padding: 3px 3px 3px 5px;
+  margin: 2.5rem 1.5rem;
+  text-decoration: none;
+  font-size: 25px;
+  color: #0e4071;
+  display: block;
+  transition: 0.3s;
+  box-shadow: 10px 10px 0 -2px #fdeaac, 10px 10px 0 0 black;
+  border-radius: 2rem;
+  background-color: #fdeaac;
+  border: 3px solid black;
+}
+
+.sidenav a:active {
+  transform: translateY(0.25rem);
+}
```

Comment:
Don't think this is needed anymore

Reply:
This actually is for when the button inside the nav is clicked